{"uid":"aaa2c6d4c6022d4c","name":"test_invalid_extra_user_roles","fullName":"tests.integration.new_relations.test_new_relations#test_invalid_extra_user_roles","historyId":"747d7c0d400591143c1e03f0a882919e","time":{"start":1714183507869,"stop":1714183507869,"duration":0},"status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 340, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 240, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 85, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 843, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 832, in _runtest_for\n    yield\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_setup\n    return (yield)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 82, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 158, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 515, in setup\n    raise exc\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 512, in setup\n    col.setup()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1775, in setup\n    self._request._fillfixtures()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 687, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 541, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 570, in _get_active_fixturedef\n    self._compute_fixture_value(fixturedef)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 647, in _compute_fixture_value\n    fixturedef.execute(request=subrequest)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1079, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 906, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 223, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 191, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"event_loop","time":{"start":1714181401427,"stop":1714181401427,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1714181401427,"stop":1714181401427,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1714181401428,"stop":1714181401558,"duration":130},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1714183507869,"stop":1714183507869,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 906, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 223, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 191, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"ops_test::finalizer","time":{"start":1714183507873,"stop":1714183508160,"duration":287},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1714183508161,"stop":1714183508161,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1714183508161,"stop":1714183508161,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1714183508161,"stop":1714183508161,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.new_relations"},{"name":"suite","value":"test_new_relations"},{"name":"host","value":"fv-az1110-321"},{"name":"thread","value":"7079-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.new_relations.test_new_relations"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":9,"skipped":9,"passed":55,"unknown":0,"total":73},"items":[{"uid":"30ec47494ba85741","reportUrl":"../2532//#testresult/30ec47494ba85741","status":"passed","time":{"start":1714098036295,"stop":1714098247937,"duration":211642}},{"uid":"c6da33572e34b9cc","reportUrl":"../2522//#testresult/c6da33572e34b9cc","status":"passed","time":{"start":1714010626951,"stop":1714010856454,"duration":229503}},{"uid":"12221a418b8a1ca7","reportUrl":"../2517//#testresult/12221a418b8a1ca7","status":"passed","time":{"start":1713924143696,"stop":1713924354412,"duration":210716}},{"uid":"cd52cc9c3340cbf5","reportUrl":"../2503//#testresult/cd52cc9c3340cbf5","status":"passed","time":{"start":1713837692702,"stop":1713837890564,"duration":197862}},{"uid":"1a62d3911c5fbd3b","reportUrl":"../2478//#testresult/1a62d3911c5fbd3b","status":"passed","time":{"start":1713751323544,"stop":1713751530047,"duration":206503}},{"uid":"9d79f6b7011146c6","reportUrl":"../2477//#testresult/9d79f6b7011146c6","status":"passed","time":{"start":1713664785901,"stop":1713664987822,"duration":201921}},{"uid":"26788c3b5cd66d8d","reportUrl":"../2473//#testresult/26788c3b5cd66d8d","status":"passed","time":{"start":1713492852686,"stop":1713493061647,"duration":208961}},{"uid":"99c6f9fca44f281","reportUrl":"../2466//#testresult/99c6f9fca44f281","status":"passed","time":{"start":1713405531650,"stop":1713405718979,"duration":187329}},{"uid":"bf15a78c2aaa3dd","reportUrl":"../2461//#testresult/bf15a78c2aaa3dd","status":"passed","time":{"start":1713319362843,"stop":1713319557745,"duration":194902}},{"uid":"f459d481b7f44898","reportUrl":"../2452//#testresult/f459d481b7f44898","status":"passed","time":{"start":1713232939685,"stop":1713233155229,"duration":215544}},{"uid":"1d28a075a86f273b","reportUrl":"../2448//#testresult/1d28a075a86f273b","status":"passed","time":{"start":1713151980926,"stop":1713152194913,"duration":213987}},{"uid":"91f2e64863635a13","reportUrl":"../2447//#testresult/91f2e64863635a13","status":"passed","time":{"start":1713060226621,"stop":1713060463662,"duration":237041}},{"uid":"1fb430ea99df21f3","reportUrl":"../2445//#testresult/1fb430ea99df21f3","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1712974145911,"stop":1712974145911,"duration":0}},{"uid":"450c2faed7c05ac2","reportUrl":"../2430//#testresult/450c2faed7c05ac2","status":"passed","time":{"start":1712887346828,"stop":1712887559172,"duration":212344}},{"uid":"3a54e4a63c2b2dca","reportUrl":"../2424//#testresult/3a54e4a63c2b2dca","status":"passed","time":{"start":1712800789186,"stop":1712801001693,"duration":212507}},{"uid":"4ed9e4dc4b423ff8","reportUrl":"../2419//#testresult/4ed9e4dc4b423ff8","status":"passed","time":{"start":1712714492555,"stop":1712714707969,"duration":215414}},{"uid":"567fb1010a94b6c7","reportUrl":"../2416//#testresult/567fb1010a94b6c7","status":"passed","time":{"start":1712628169124,"stop":1712628379423,"duration":210299}},{"uid":"26de6ec3cecc19d","reportUrl":"../2409//#testresult/26de6ec3cecc19d","status":"passed","time":{"start":1712541718386,"stop":1712541923485,"duration":205099}},{"uid":"dc6c6442d5e23869","reportUrl":"../2408//#testresult/dc6c6442d5e23869","status":"passed","time":{"start":1712455264896,"stop":1712455478115,"duration":213219}},{"uid":"9fafaa382dd0ec45","reportUrl":"../2407//#testresult/9fafaa382dd0ec45","status":"passed","time":{"start":1712368752489,"stop":1712368974231,"duration":221742}}]},"tags":["asyncio","group(1)"]},"source":"aaa2c6d4c6022d4c.json","parameterValues":[]}